src/kili/entrypoints/mutations/issue/__init__.py

```diff
@@ -10,11 +9,11 @@
 from kili.core.helpers import deprecate
 from kili.entrypoints.base import BaseOperationEntrypointMixin
 from kili.entrypoints.mutations.asset.helpers import get_asset_ids_or_throw_error
+from kili.gateways.kili_api_gateway.issue.operations import GQL_CREATE_ISSUES
```

Comment:
to remove

Reply:
it is still used in the 2 other mutations that we didn't move: create_questions and append_to_issues